local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Mar 29 09:34:41 PYT 2019
+sdk.dir=/home/rodrigo/Android/Sdk
```

Comment:
This file is created by Android studio automatically; its for local use only. Please remove it from the git-tracked files.